app/src/main/java/com/framgia/wsm/screen/login/LoginPresenter.java

```
@@ -122,4 +203,27 @@ private void validatePasswordInput(String password) {
             mViewModel.onInputPasswordError(messagePassword);
         }
     }
+
+    private String getTime() {
+        DateFormat df = new SimpleDateFormat("yyyy-MM-dd");
+        String date = df.format(Calendar.getInstance().getTime());
+        return date;
+    }
+
+    private double getDayOff(String day) {
+        Date date1;
+        Date date2;
+        double dayDifference = 0;
+        SimpleDateFormat dates = new SimpleDateFormat("yyyy-MM-dd");
```

WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.